browser/resources/settings/brave_sync_page/brave_sync_code_dialog.html

```diff
@@ -333,12 +333,12 @@ <h2 class="device-title">$i18n{braveSyncChooseDeviceComputerTitle}</h2>
       <textarea autofocus class="sync-code-text" value="{{syncCode::input}}"></textarea>
       <div class="sync-code-footer">
         [[i18n('braveSyncWordCount', syncCodeWordCount_)]]
-        <template is="dom-if" if="[[isInvalidSyncCode]]">
-          <div class="invalid-sync-code">
-            $i18n{braveSyncInvalidSyncCodeTitle}
-          </div>
-        </template>
       </div>
+      <template is="dom-if" if="[[isInvalidSyncCode]]">
```

Comment:
Seems like we can now remove isInvalidSyncCode as it isn't being used for anything else, and also seems like this new property `syncCodeValidationResult` is only used for an error. No need to have 2 indicators of an error, so maybe let's combine to a single property - `syncCodeValidationError` which can be a string, and display this output if it's non-null?

Also what was the readon for moving it out of the footer? Did it still look ok?

Reply:
About 1st notice - yes, this makes sense, I am re-doing this to have only `syncCodeValidationError`, indeed, it should be enough to have just one var, thanks.

About 2nd question - having error text inside the footer caused the number of words field be flattened, so I moved the error block. Here are the screenshot
| Before  | After |
| ------------- | ------------- |
|![Screenshot from 2022-04-26 14-07-08](https://user-images.githubusercontent.com/24739341/165293483-47089675-d1e5-4388-bb43-b568c5fde66f.png)|![Screenshot from 2022-04-26 13-58-31](https://user-images.githubusercontent.com/24739341/165293547-8abd3de4-1e52-4968-8a17-5aa0b164b7bb.png)|

Reply:
this is changed in https://github.com/brave/brave-core/pull/12300/commits/f15a93b30eda33bdbfda9148787eece378cb17f3